README.md

```
@@ -502,25 +502,88 @@ sudo cat /var/log/aws-routed-eni/ipamd.log | tail -100
 The deployment role has:
 
-- **PowerUserAccess** - Full access to AWS services
-- **Scoped IAM** - Manage roles, policies, instance profiles, OIDC providers (excludes users/groups)
+- **PowerUserAccess** - Full access to AWS services (managed policy)
+- **Scoped IAM** - Inline policy to manage roles, policies, instance profiles, OIDC providers (excludes users/groups)
 
-**Assuming the deployment role:**
+#### How It Works
+
+**CI/CD (GitHub Actions):**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)